### ISSUE TITLE
Model for RDF graph with participant node

### ISSUE THREAD

**Issue by smpadhy** [ReproNim/brainverse#99]
Currently, a plan, say Plan_1, has several attributes <ID,NumberOfSession,Name,Description, SessionCollections>.  A Session Collection has members <Session1, Session2, ...>. Each Session has  attributes<name, Instrument collections >. Instrument Collection has members <Instrument1, instrument2, ...>. An instrument has attributes <task name, description, assignee, time estimate, instrument name>.

While adding a participants node, it should be added to the plan, Plan_1, as another attribute with 'wasAssociatedWith' predicate. Since there can be several participants for an experiment plan,  corresponding to each participant a node will be added to the RDF Plan graph.

Another alternative could be to have a participants Collection associated to the plan and each participant could be member of that collection.

Another question - should participant be part of plan or part of data collection activity? As part of plan, there might be prescreen session to recruit participants.

**Comment by smpadhy** [comment]
@satra @dbkeator @khelm Any thoughts?

**Comment by khelm** [comment]
Hi @smpadhy 
We've decided to use Subject rather than Participant in NIDM-Experiment [see here](https://github.com/incf-nidash/nidm/issues/425). 

We should also talk about the number of Collections to be created - on an earlier discussion on a NIDM call the feeling was to minimize the number of collections. (I do include several in the ontology though).
 
I would keep `prov:Plan` as not directly involving Subjects.  There is the term "Group" in the ontology that is used for collections of `prov:Person`'s who have `prov:role nidm:Subject` and are associated with `nidm:DataAcquisition`.

The prescreening of subjects would be a separate activity, with two Agents: the person who administered the prescreening and the Subject. It would look the same as the InformedConsent activity in the latest version of the lucidchart in the NIDASH folder.

**Comment by satra** [comment]
since collections can be built from the underlying graph, i would rather avoid collections when possible.

**Comment by smpadhy** [comment]
@khelm @satra Thank you for the clarifications.
@khelm In the NIDASH/Specifications folder, which is the latest lucidchart version? I am unable to open NIDM-Experiment-Hierarchy.

**Comment by khelm** [comment]
I agree that Collections should be minimized, but I am not totally against them.  For example, one could envision a collection of "all dicom tags for this AcquisitionObject" or an "AuxiliaryFile" collection that would include the b-val and b-vec files.  Yes, you could build them each time from the underlying graph each time, but there is something attractive about it conceptually.

The lucid chart you want is NIDM-Experiment_Draft_8a in the NIDASH folder
